server/svix-server/migrations/20231010170520_messagecontent_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE messagecontent (
+    id character varying NOT NULL COLLATE pg_catalog."C",
+    created_at timestamp with time zone NOT NULL,
+    payload bytea NOT NULL,
```

Comment:
You mentioned in the PR description you chose `bytea` for future flexibility. Is there some advantage over just using a text field since JSON is utf-8 (I thought)?

Reply:
The possibility is that we could actually store binary data in the future, hence the binary type.